### DISCUSSION TITLE
Customizer : Color section

### DISCUSSION THREAD

**Discussion by luukee** [wprig/wprig#791] [Q&A]
Hi, love working with the theme :)

I was wondering how we add new color modules to the "Color" section on the customizer? Might be super obvious, but I couldn't seem to find it in `inc/EZ_Customizer`, `assets/js/src/customizer.js`, or `themeCustomizeSettings.json`.

![Screen Shot 2022-03-04 at 6 05 09 PM](https://user-images.githubusercontent.com/30611098/156863498-5c0e9d32-9e56-43bc-bd04-0bc7d1364976.png)

That would be super helpful! Thanks, 
Luke

**Comment by luukee** [comment]
@mor10 Are you still around on this project? 

I always loved watching your video tutorials on Lynda back in the day :grinning:. Would really love to "Get Cracking" once I have an answer to this question :wink:. Either way, you are a hero of mine, to say the least.

Thanks!
- Luke

**Comment by robruiz** [comment]
Morten no-longer maintains this project. However, it still stays true to
his vision. We are always looking for contributors and maintainers.

**Reply by luukee** [reply]
Great @robruiz! Thanks for jumping in. Glad you are keeping up with it.

On that note, did you see my original question?

> I was wondering how we add new color modules to the "Color" section on the customizer? Might be super obvious, but I couldn't seem to find it in `inc/EZ_Customizer`, `assets/js/src/customizer.js`, or `themeCustomizeSettings.json`.

![156863498-5c0e9d32-9e56-43bc-bd04-0bc7d1364976](https://user-images.githubusercontent.com/30611098/159837528-bf4d301b-4517-48b3-a05e-69ce2ad414b4.png)

Thanks again.

**Reply by robruiz** [reply]
Did you ever end up getting this figured out? Also, if you have questions, one of the best places to ask them is in the WP Rig Slack. To get in there, go here: https://join.slack.com/t/wprig/shared_invite/zt-1j92phnpu-wPTBnsTzki7S1YS~lqh6Ug

**Reply by luukee** [reply]
@robruiz I never figured out how to alter the "Colors" section. But instead, I created a new section for custom colors. Thanks for checking in on it!

**Comment by robruiz** [comment]
Use the EZ _customizer class instead.
